code/modules/mob/living/basic/guardian/guardian_fluff.dm

```diff
@@ -0,0 +1,124 @@
+	guardian.attack_verb_continuous = attack_verb_continuous
+	guardian.attack_verb_simple = attack_verb_simple
+	guardian.attack_sound = attack_sound
+	guardian.attack_vis_effect = attack_vis_effect
+
+	guardian.overlay = mutable_appearance(guardian.icon, overlay_state)
+
+/// Output an appropriate fluff string for our guardian when it is created
+/datum/guardian_fluff/proc/get_fluff_string(guardian_type)
+	return span_holoparasite(guardian_fluff[guardian_type] || "You bring forth a glitching abomination, something which should not be! Please contact a coder about it.")
+
+/// Used by holoparasites in the Traitor uplink
+/datum/guardian_fluff/tech
+	name = "Holoparasite"
+	fluff_type = GUARDIAN_TECH
+	bubble_icon = "holo"
+	icon_state = "techbase"
+	overlay_state = "tech"
+	guardian_fluff = list(
+		GUARDIAN_ASSASSIN = "Boot sequence complete. Stealth modules loaded. Holoparasite swarm online.",
+		GUARDIAN_CHARGER = "Boot sequence complete. Overclocking motive engines. Holoparasite swarm online.",
+		GUARDIAN_DEXTROUS = "Boot sequence complete. Armed combat routines loaded. Holoparasite swarm online.",
+		GUARDIAN_EXPLOSIVE = "Boot sequence complete. Payload generator online. Holoparasite swarm online.",
+		GUARDIAN_GASEOUS = "Boot sequence complete. Atmospheric projectors operational. Holoparasite swarm online.",
+		GUARDIAN_GRAVITOKINETIC = "Boot sequence complete. Gravitic engine spinning up. Holoparasite swarm online.",
+		GUARDIAN_LIGHTNING = "Boot sequence complete. Tesla projectors charged. Holoparasite swarm online.",
+		GUARDIAN_PROTECTOR = "Boot sequence complete. Bodyguard routines loaded. Holoparasite swarm online.",
+		GUARDIAN_RANGED = "Boot sequence complete. Flechette launchers operational. Holoparasite swarm online.",
+		GUARDIAN_STANDARD = "Boot sequence complete. CQC suite engaged. Holoparasite swarm online.",
+		GUARDIAN_SUPPORT = "Boot sequence complete. Medical suite active. Holoparasite swarm online.",
+	)
+
+/// Used by powerminers found in necropolis chests
+/datum/guardian_fluff/miner
+	name = "Power Miner"
+	icon_state = "minerbase"
+	overlay_state = "miner"
+	guardian_fluff = list(
+		GUARDIAN_ASSASSIN = "The shard reveals... Glass, a sharp but fragile ambusher.",
+		GUARDIAN_CHARGER = "The shard reveals... Titanium, a lightweight, agile fighter.",
+		GUARDIAN_DEXTROUS = "The shard reveals... Gold, a malleable hoarder of treasure.",
+		GUARDIAN_EXPLOSIVE = "The shard reveals... Gibtonite, volatile and surprising.",
+		GUARDIAN_GASEOUS = "The shard reveals... Plasma, the bringer of flame.",
+		GUARDIAN_GRAVITOKINETIC = "The shard reveals... Bananium, a manipulator of motive forces.",
+		GUARDIAN_LIGHTNING = "The shard reveals... Iron, a conductive font of lightning.",
+		GUARDIAN_PROTECTOR = "The shard reveals... Uranium, dense and resistant.",
+		GUARDIAN_RANGED = "The shard reveals... Diamond, projecting a million sharp edges.",
+		GUARDIAN_STANDARD = "The shard reveals... Platinum, a powerful fighter.",
```

Comment:
whyd you rename it from adamantine, platinum isnt a material in ss13

Reply:
![image](https://github.com/tgstation/tgstation/assets/51863163/18de3714-0165-4881-af68-f975e38900f0)

Reply:
then youd rename ranged to hierophant bro

Reply:
rename it to Plastitanium it's close enough